src/Module/Notifications/Introductions.php

```diff
@@ -82,34 +81,34 @@ public static function content(array $parameters = [])
 	
 		// Loop through all introduction notifications.This creates an array with the output html for each
 		// introduction
-		/** @var Introduction $notification */
-		foreach ($notifications['notifications'] as $notification) {
+		/** @var Introduction $Introduction */
```

Comment:
tbh I'm still against breaking the camelCase rule for objects. I think this will lead to heterogeneous usage because you have to know what kind of type you're creating. And I don't see any reason for it, because we use the same naming-convention for the class names itself .. I can live with it, but I don't find it intuitive :X

Reply:
It's a convention I started using while I was refactoring arrays into objects. I would change the type of a parameter or a variable from array to Entity/Collection in one part of a file, and then I would have make sure that all the related array key accesses were converted to object property accesses. So `$notification['key']` could still exist in the rest of the file while I was doing the refactoring, but `$Notification['key']` needed to be updated to `$Notification->key`.

Reply:
aaahh now I understand .. does that mean in case there aren't any arrays left, you would camelCase the objects as well again? Than I fully agree with it :)

Reply:
Absolutely. During the transition I need some sort of visual cue but once we get rid of arbitrary arrays there's no reason to keep the capitalized versions.